Update Japanese translation(1.16)

koh-gh opened the pull request:
- Added new translation
- Unified the disjointed translations
- Changed the translation of 'Manyullyn'

koh-gh committed:
Update ja_jp.json

koh-gh committed:
Update ja_jp.json

koh-gh committed:
Update ja_jp.json

koh-gh committed:
Update ja_jp.json

karakufire commented on src/main/resources/assets/tconstruct/lang/ja_jp.json:
```diff
@@ -1,1108 +1,1181 @@
+
+  "block.tconstruct.sky_slime_fluid": "液状スカイスライム",
+  "fluid.tconstruct.sky_slime": "液状スカイスライム",
+  "fluid.tconstruct.flowing_sky_slime": "液状スカイスライム",
```
「流れる」を語頭につけ忘れているように見受けられます

karakufire commented on src/main/resources/assets/tconstruct/lang/ja_jp.json:
```diff
@@ -1,1108 +1,1181 @@
+
+  "block.tconstruct.molten_uranium_fluid": "溶融したウラン",
+  "fluid.tconstruct.molten_uranium": "溶融したウラン",
+  "fluid.tconstruct.flowing_molten_uranium": "溶融したウラン",
```
「流れる」を語頭につけ忘れているように見受けられます

koh-gh committed:
Update ja_jp.json

Fixed some missing translation

koh-gh commented:
更新しました。コメントありがとうございました。

koh-gh committed:
Update ja_jp.json

Tweaked translation

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Update ja_jp.json

- Changed unarmed translation
- Reflected the changes of en_us.json.

koh-gh committed:
Update ja_jp.json

Fixed spilling translation

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Updade Japanese translations

Tweaked ja_jp.json
Added book translation

koh-gh commented:
Since I added the translations of the book, I changed the title.

koh-gh committed:
Update Japanese translation

Add book pages(dyed, embellishment)
Fixed some incorrect traveler's gear translation

CKenJa commented on src/main/resources/assets/tconstruct/lang/ja_jp.json:
```diff
@@ -1,1108 +1,1174 @@
-  "block.tconstruct.pig_iron_block.tooltip": "ビーコンベース可能",
-  "item.tconstruct.pig_iron_ingot": "豚鉄インゴット",
-  "item.tconstruct.pig_iron_nugget": "豚鉄塊",
+  "block.tconstruct.pig_iron_block": "銑鉄ブロック",
```
銑鉄はそのままピッグアイアンのほうがわかりやすいかと
1.12.2のときは豚鉄だった気がしますし

koh-gh replied:
確かにTiCではPig Ironに豚のイラストを使っていてその意見も理解できるのですが、実際「Pig Iron」の日本語は「銑鉄」であり、ネットを検索すればなぜ「銑鉄」が英語で「Pig Iron」と呼ばれているのか複数のサイトで解説されています。このため、今回は正確な日本語である「銑鉄」にしています。
なぜ「銑鉄」のイラストに豚が使われているのか興味を持つ人が出れば、英語を学ぶ機会にもなるかなと考えています。

koh-gh replied:
I think "pig iron" is not a fictitious metal like "pig-infused iron", but real metal. And piggy textures are joke.

CKenJa commented on src/main/resources/assets/tconstruct/lang/ja_jp.json:
```diff
@@ -1,1108 +1,1174 @@
-  "block.tconstruct.cobalt_ore": "ネザーラックのコバルト原石",
-  "block.tconstruct.cobalt_block": "コバルトのブラック",
-  "block.tconstruct.cobalt_block.tooltip": "ビーコンベース可能",
+  "block.tconstruct.cobalt_ore": "ネザーコバルト原石",
```
raw oreではないので原石ではなく鉱石が適切かと

koh-gh replied:
修正漏れでした。ありがとうございました。

koh-gh committed:
Update ja_jp.json

Fixed nether cobalt ore translation

koh-gh committed:
Update book translation

Tweaked book translation

koh-gh committed:
Update translation

Update for https://github.com/SlimeKnights/TinkersConstruct/issues/4720
- Added "name" section to all items in "sections" files
- Added substitution definition to language.json

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Update translation

Backported corrections of errors noticed while working on the 1.18 translation.

koh-gh committed:
Update translation

Tweaked translation

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Merge remote-tracking branch 'upstream/1.16' into 1.16

koh-gh committed:
Update Japanese translation

Reflected changes in 3.3.4.335

KnightMiner commented:
This translation looks relatively up to date. Are you available to get the last few new features for 1.16.5 in? If so we can merge this in for one last 1.16 release with updated translations. I can merge it in without updates, but would be nice for it to be complete

koh-gh commented:
This translation has been confirmed to work with 3.3.4.335, so you can merge it as it is.

koh-gh commented:
There seems to be a dev build 1.16.5-3.3.4.336, are there any fixes that affect the translation? If not, this will work.

KnightMiner commented:
Probably just an automatically made build when Jenkins saw I pushed commits, nothing that is not on curseforge. That said, your changes last month will not include the new features I added this week (couple backports of 1.18 features). Like I said, I can merge it without updates, it will mostly work. Would just be nice to have the new features.

koh-gh committed:
Update translation

Updated diamond modifier description on Encyclopedia.

koh-gh commented:
This includes darkthread, ancient hide, 4 embellishments. Please see https://github.com/SlimeKnights/TinkersConstruct/pull/4707/commits/d81b1c57355d2f2487f657c18e11b908fb7409dc. GitHub shows it as last month, but it was updated yesterday.
But diamond modifier update was lacked. Updated this.

KnightMiner commented:
Huh, weird. Must be a github bug. Need to do one last test, so should be able to merge it in some time this week.

KnightMiner commented:
I am merging this to ensure if we do another 1.16 release its included, but at this time another 1.16 release is not guaranteed.
